erp-client/src/app/app.module.ts

```diff
@@ -1,45 +1,119 @@
-import {BrowserModule} from '@angular/platform-browser';
-import {NgModule} from '@angular/core';
-import {MaterialModule} from './material.module';
-import {AppRoutingModule} from './app-routing.module';
+// Angular Components
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
+import { NgModule } from '@angular/core';
 
-import {AppComponent} from './app.component';
-import {LoginComponent} from './components/login/login.component';
-import {FormsModule} from '@angular/forms';
-import {HomeComponent} from './components/home/home.component';
-import {HttpClientModule} from '@angular/common/http';
+// Root Directory Components
+import { AppComponent } from './app.component';
+import { AppRoutingModule } from './app-routing.module';
+import { MaterialModule } from './material.module';
+
+// Page Components
+import { AdminControlsComponent } from './components/admin-controls/admin-controls.component';
+import { AdminTableComponent } from './components/admin-table/admin-table.component';
+import { AwardComponent } from './components/award/award.component';
 import { CanvasComponent } from './components/canvas/canvas.component';
+import { CreateAwardComponent } from './components/award/createAward.component';
+import { HomeComponent } from './components/home/home.component';
+import { LoginComponent } from './components/login/login.component';
 import { SelfComponent } from './components/self/self.component';
-import { AdminTableComponent } from './components/admin-table/admin-table.component';
-import { AdminControlsComponent } from './components/admin-controls/admin-controls.component';
-import {MatCardModule, MatIconModule, MatProgressSpinnerModule, MatSidenavModule, MatListModule, MatToolbarModule} from '@angular/material';
+
+// Angular Material Components
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { MatButtonModule } from '@angular/material';
```

Comment:
I know we won't end up using all of these, but I figured we could just include them all now, and then we can prune this to get rid of whatever we don't use once we're done.

Reply:
Also, since there's a lot in there right now, I sort of compartmentalized it a bit and sorted the compartments alphabetically to make it easier to check on what's there and what's not.

Reply:
I like the idea of functional separation for the imports. There's a 'material.module.ts' file that already has some of the material imports if we want to move the rest of them there. Referenced in this import statement:
`import { MaterialModule } from './material.module';`

Reply:
Good call. I'll move all the Material stuff to ```'./material.module'```